### PR TITLE
Build /ai/edge page

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/17eHoiLrhFLrfV8UXD_4bxn-yY1_YToxf4cs8-5kYkxo/edit) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=125-8552&m=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1306.demos.haus/solutions/ai/edge
- Check against doc and design
- Check layout on smaller screens

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12553

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
looks good, jkust 2 things:

This heading violates the content limits we're working on. Any chance the part after the hyphen could go in a separate h2 in the right hand side? cc @Sophie-32 
image


no need for p-section shallow on the last elements - there's already enough spacing due to the p-section class at the top oof each section:
image

**Comment by Sophie-32** [comment]
@mtruj013 - left a quick note in the copydoc

**Comment by Sophie-32** [comment]
> * https://canonical-com-1306.demos.haus/solutions/ai/edge

we're going to have to let it go this time, there were a lot of stakeholders and it would hold things up to try and redo copy. It's something I'm talking to Freyja about.

File: templates/solutions/ai/edge.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-suru--50-50 p-section--shallow">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-section--shallow">
+          Develop AI models and run
+          <br />
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:23]:
```suggestion
          <br class="u-hide--medium u-hide-small" />
```

File: templates/solutions/ai/edge.html [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+          them at the edge &mdash; all on the
+          <br />
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:25]:
```suggestion
          <br class="u-hide--medium u-hide-small" />
```

File: templates/solutions/ai/edge.html [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        </h1>
+      </div>
+      <div class="col">
+        <p>
+          Edge computing and AI go hand-in-hand to drive some of today's most valuable enterprise use cases. Canonical delivers a single stack with everything you need to easily and securely deploy and manage AI at the edge. Unlock the power of real-time, AI-driven insight in the field.
+        </p>
+        <hr />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="ai-modal">Contact Canonical</a>
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:36]:
```suggestion
           aria-controls="ai-modal">Contact Canonical <span class="u-off-screen">about AI solutions for edge computing</span></a>
```
This is something I have started to do on links (after you told me about it haha). It isn't necessary but I think adds a nice QOL for people using screen readers, even when there is context

File: templates/solutions/ai/edge.html [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+        </p>
+        <p>
+          By centralising your edge AI journey on a Canonical stack, you can mitigate the complexity of AI projects that rely on a wide array of open source tools. Keep your projects on track and ensure they deliver ROI in production.
+        </p>
+        <p>
+          Canonical's portfolio encompasses the complete edge AI toolset, including the OS, Kubernetes, MLOps, compute, data processing and certified hardware &mdash; all optimised for the edge.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/engage/ai-on-the-edge">Watch our AI at edge webinar&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:75]:
```suggestion
          <a href="https://ubuntu.com/engage/ai-on-the-edge">Watch our AI at the edge webinar&nbsp;&rsaquo;</a>
```

File: templates/solutions/ai/edge.html [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-suru--50-50 p-section--shallow">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-section--shallow">
+          Develop AI models and run
+          <br />
+          them at the edge &mdash; all on the
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:24]:
```suggestion
          them at the edge &mdash; all on the
```
Usually in UK English we opt for the ndash, but not a biggie

File: templates/solutions/ai/edge.html [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+      <p class="p-heading--2">
+        <a href="/contact-us" aria-controls="ai-modal">Contact us&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </section>
+
+  <script defer src="{{ versioned_static('js/modals.js') }}"></script>
+
+  {% with %}
+    {% set returnURL = "https://canonical.com/solutions/ai#contact-form-success" %}
```
Comment (petesfrench) [inline templates/solutions/ai/edge.html:268]:
```suggestion
    {% set returnURL = "https://canonical.com/solutions/ai/edge#contact-form-success" %}
```
Should this not return to the same page?

**Review by petesfrench — APPROVED** [review]
LGTM, just a couple minor changes.
Sorry it got split up weird, I clearly still don't know how to use github haha

File: templates/solutions/ai/edge.html (hunk 3 above)
Comment (mtruj013) [reply templates/solutions/ai/edge.html:36]: Agreed, thanks for flagging!

File: templates/solutions/ai/edge.html (hunk 5 above)
Comment (mtruj013) [reply templates/solutions/ai/edge.html:24]: I'm never confident about these tbh, but if this is norm in the UK then I'll use this from now on 🙂

File: templates/solutions/ai/edge.html (hunk 6 above)
Comment (mtruj013) [reply templates/solutions/ai/edge.html:268]: It very much should, nice catch!

**Pull request merged**